### DISCUSSION TITLE
Feature Request: page description on-page

### DISCUSSION THREAD

**Discussion by elizabethyalkut** [colinwilson/lotusdocs#111] [Feature Requests]
If a page has a description in the front matter, I'd like to be able to see that text in the page itself. Often my descriptions on list pages get truncated, but contain valuable contextualizing information for the pages contained within the section. (I do not control the length of content, and some languages take many more characters to express a given concept than English does.)

This could be either a parameter that allowed me to set whether descriptions appear on the page for the whole site or a per-page `subtitle` front matter item so I could pick and choose which pages display the contextualizing information.

**Comment by colinwilson** [comment]
Sorry for the delay in response. 

> Often my descriptions on list pages get truncated

The truncation limit for list page descriptions is controlled by the `.Site.Params.docs.listDescTrunc` parameter in your `hugo.toml` config file. 

Is this helpful?

> set whether descriptions appear on the page for the whole site or a per-page

Do you mean for list page descriptions to appear on the pages nested inside?

**Reply by elizabethyalkut** [reply]
Unfortunately, increasing the truncation limits is not a great solution, because of the difference in what is a reasonable truncation limit for English versus, say, Hebrew.

Yes, I'd like to be able to see the front matter "description" string on a single page, which currently appears only on the list page, on the page itself. Does that clarify?
